packages/uxpin-merge-cli/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [3.0.1] - 2023-02-15
+
+### Added
+
+- Adds `Access-Control-Allow-Private-Network: true` in CROS Response ([#369](https://github.com/UXPin/uxpin-merge-tools/pull/369))
+
```

Comment:
Instead of `added`, I think `fixed` would make more sense as we are not adding a new feature?

Something like this, to focus more on the problem we solve:

```
Fixed:

- Adds header in experimental mode to fix CORS error in Chrome (...)
```

Reply:
Fixup the CHANGELOG.md in [here](https://github.com/UXPin/uxpin-merge-tools/commit/6224aa8b7b863d8c8712b0296b76cd20d51212ea)